Zend/zend_compile.c

```diff
@@ -7524,6 +7524,16 @@ static void zend_compile_func_decl(znode *result, zend_ast *ast, bool toplevel)
 		}
 
 		zend_compile_attributes(&op_array->attributes, decl->child[4], 0, target, 0);
+
+		zend_attribute *override_attribute = zend_get_attribute_str(
+			op_array->attributes,
+			"override",
```

Comment:
I am aware. The macro is incredibly rarely used and it is indeed very weird by including the comma itself.

Zend/* is not my main area of expertise, is this remark (a) a request for change or (b) a simple suggestion / heads up?

Reply:
We're inconsistent in the usage, it's not required so I'll let it up to you. In the usual macro fashion it can be dangerous, i.e. `(ZEND_STRL("foo"))` expands to `("foo", ("foo", sizeof("foo") - 1))` which is the comma operator in C (https://en.wikipedia.org/wiki/Comma_operator). If one is unaware of this macros that might be confusing.

Reply:
I'll leave it as is then, it's consistent with `#[\ReturnTypeWillChange]` at:

https://github.com/php/php-src/blob/ee4ebab5a979c3245a7c3c257e9b123606a5793c/Zend/zend_inheritance.c#L1026-L1030